feg/gateway/services/s6a_proxy/servicers/s6a_proxy.go

```diff
@@ -35,7 +35,7 @@ import (
 
 const (
 	ULR_RAT_TYPE     = 1004
-	ULR_FLAGS        = 1<<1 | 1<<5
+	ULR_FLAGS        = 1<<1 | 1<<5 // 29.272 Table 7.3.7/1: ULR-Flags (S6a/S6d-Indicator, Initial-AttachIndicator)
```

Comment:
0b100010 could be easier to read - https://play.golang.org/p/oSavbH6XtbO

Reply:
I agree. I thought shifting bits was the `magma standard`
I will modify it